.golangci.yaml

```diff
@@ -5,42 +5,72 @@ run:
   tests: true
   skip-dirs:
     - vendor
-    - .go
+  skip-dirs-use-default: false
+
 
 output:
   format: colored-line-number
   print-issued-lines: true
   print-linter-name: true
-
+issues:
+  exclude-use-default: false
+  exclude:
+    # should have a package comment, unless it's in another file for this package (golint)
+    - 'ST1000: at least one file in a package should have a package comment'
 linters-settings:
+  govet:
+    # Enable all analyzers.
+    # Default: false
+    enable-all: true
+    # Disable analyzers by name.
+    # Run `go tool vet help` to see all analyzers.
+    # Default: []
+    disable:
+      - fieldalignment # too strict
+    # Settings per analyzer.
+    settings:
+      shadow:
+        # Whether to be strict about shadowing; can be noisy.
+        # Default: false
+        strict: true
+
+
   gocyclo:
     # minimal code complexity to report, 30 by default (but we recommend 10-20)
     min-complexity: 30
-  govet:
-    check-shadowing: true
   goconst:
     min-len: 3
     min-occurrences: 4
+  decorder:
+    # Required order of `type`, `const`, `var` and `func` declarations inside a file.
+    # Default: types before constants before variables before functions.
+    dec-order:
+      - type
+      - const
+      - var
+      - func
 
 linters:
   disable-all: true
   enable: # NOTE: please keep this list alphabetically sorted
     - bodyclose
     - deadcode
+    - decorder
     - dogsled
-    - errcheck
+    #- errcheck
```

Comment:
why disable this?

Reply:
it was not working before, but with this fix now turning  on that will generate many fixed where we just ignore the  error.